Configuration/ProcessModifiers/python/allSonicTriton_cff.py

```diff
@@ -0,0 +1,6 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.ProcessModifiers.enableSonicTriton_cff import enableSonicTriton
+
+# collect all SonicTriton-related process modifiers here
+allSonicTriton = cms.ModifierChain(enableSonicTriton)
```

Comment:
This does not prevent the use of customization functions where desired. However, there may be other modifiers required as well (e.g. the `mlpf` modifier in #32048). The goal here is to facilitate a single Triton-enabled special workflow, with all available producers included. (Customization functions can also be included in the workflow definition directly, or via `makeProcessModifier()`.)

Reply:
How would the Modifier be used then? Within production configuration fragments, non-production configurations fragments, customize functions, something else?

Reply:
Right now, this `allSonicTriton` modifier is just intended to simplify the special workflow setup in `Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py`.

I could envision a similar aggregating customize function like `customizeAllSonicTriton` to aggregate any and all customize functions, and then the special workflow would just need that one function. That isn't introduced here because there's nothing to test it with (yet).

Reply:
I'm still a bit confused by the purpose of these two Modifiers (now). In #32272 the motivation to favor customize functions was to avoid putting customization for "R&D and testing"  into production configuration fragments. If the behavior is customized with a function, I don't see the added value of centrally defined Modifiers.

Reply:
For example:
#32048 introduces a modifier `mlpf`† to activate the ML ParticleFlow. There is also a SonicTriton version of the MLPF producer that will be introduced. Even if that producer is activated via a customize function, the `mlpf` modifier will still be needed to include it in a workflow. Therefore, it is worthwhile to be able to aggregate multiple relevant modifiers for a SonicTriton-enabled workflow.

† because this "policy" of using customize functions for R&D was just invented last month to placate certain partisans and is rarely applied anywhere else, but I digress.